Migrate Validator/PM role

ramyaragupathy opened the pull request:
From #2474 

*  Create `undefined` organisation
*  Create a validator team - `undefined-validators`
*  Create a PM team - `undefined-pms`
*  Populate these teams with current validators and project managers
*  Set all existing validator and project manager role to mapper
*  ~Update key method calls to include a fallback option for the `undefined-validators` and `undefined-pms`~
* Map the new validator team to all projects with validation permission set to TEAMS, TEAMS_LEVEL

On downgrade:
*  Disassociate all projects and validator team relations
*  Update users role to validators and project managers based on data present in respective teams
*  Delete teams and orgs

xamanu commented:
Looks good so far. Can we include the deprecation of validator and pm roles in TM4?

xamanu commented:
I rebased on `develop` and the migrations were referencing a migration that didn't exist. Put that on the lastest migration.

ramyaragupathy commented:
I've made more changes to backend:
* remove the `is_user_a_project_manager`. In earlier it just had admin check making it equivalent to `is_user_an_admin`. So this function is repetitive and hence no more available for use
* Updated unit test cases consuming `is_user_a_project_manager` function
 
@willemarcel - can you support on frontend changes to remove PM and validator roles please? I attempted a few changes on my side but ended up with an `iD` not found error. Here are the files I updated:

* `frontend/src/components/header/tests/menuItens.test.js`
* ` frontend/src/components/projectEdit/messages.js`
* ` frontend/src/components/teamsAndOrgs/messages.js`
* `frontend/src/components/user/messages.js`
* `frontend/src/locales/en.json`
* `frontend/src/locales/he.json`
* `frontend/src/locales/id.json`
* `frontend/src/locales/pt_BR.json`
